packages/opentelemetry-node/test/registration.test.ts

```diff
@@ -58,28 +56,29 @@ describe('API registration', () => {
   it('should register configured implementations', () => {
     const tracerProvider = new NodeTracerProvider();
 
-    const contextManager = new NoopContextManager();
-    const propagator = new NoopTextMapPropagator();
+    const mockContextManager = { disable() {} } as any;
```

Comment:
wouldn't be safer to have `ContextManager` type  here instead of any?

Reply:
and the same for with `TextMapPropagator`

Reply:
its just a test mock and it doesn't satisfy the whole interface. the types really don't matter on it. It gets disabled and removed at the end of the test anyway. It's just enough function to get the tests to pass (they call disable between each test)